includes/controllers/pages/class-show-category.php

```diff
@@ -15,7 +15,8 @@ public function dispatch() {
         $html = '';
 
         if ( is_object( $term ) ) {
-			if ( WPBDP__Themes_Compat::is_block_theme() ) {
+			if ( is_callable( 'WPBDP__Themes_Compat::s_block_theme' ) && WPBDP__Themes_Compat::is_block_theme() ) {
```

Comment:
Hey @stephywells it should be `is_callable( 'WPBDP__Themes_Compat::is_block_theme' )`

Reply:
@theRealRizeo thanks so much for letting me know! I really appreciate your help!